vault/activity_log.go

```diff
@@ -2289,29 +2316,29 @@ func (a *ActivityLog) segmentToPrecomputedQuery(ctx context.Context, segmentTime
 		if segmentTime == opts.activePeriodEnd {
 			a.metrics.SetGaugeWithLabels(
```

Comment:
should a metric be added for secret sync assoc as well? Or is it gonna be in another PR?

Reply:
that will be in a later PR